pyproject.toml

```diff
@@ -40,6 +40,7 @@ dev = [
     "pre-commit>=4.3.0",
     "pytest>=8.4.2",
     "pytest-cov>=7.0.0",
+    "qdrant-client>=1.15.1",
```

Comment:
This is currently installed as a dev dependency, but isn't it a required dependency?

Reply:
I thought it went under dev dependencies since this was more exploratory instead of a feature, but I was mistaken. Updated.